packages/server/src/common/server.ts

```diff
@@ -40,10 +40,17 @@ class ExpressServer {
             maxAge: '30d',
             immutable: true,  // these files never change and can be cached for 1y
         }));
-        app.use(express.static(`${clientRoot}/build`, {
+
+        // this must be set before the root /build directory or it will not take effect
+        app.use('/index.html', express.static(`${clientRoot}/build/index.html`, {
             maxAge: '1m', // index.html should be revalidated much more frequently
         }));
 
+        // if we change these files we'll have to invalidate the cache in gcp console
+        app.use(express.static(`${clientRoot}/build`, {
```

Comment:
favico, logo, manifests don't change, we can invalidate in gcp console if we ever do